crates/prose-xmpp/src/stanza/vcard4.rs

```diff
@@ -24,6 +26,10 @@ pub struct VCard4 {
     pub tel: Vec<Tel>,
     pub title: Vec<Title>,
     pub url: Vec<URL>,
+    /// Unsupported standard properties, defined in [RFC 6350 - vCard Format Specification, section 3.3](https://datatracker.ietf.org/doc/html/rfc6350#section-3.3) or by IANA (see [vCard Elements](https://www.iana.org/assignments/vcard-elements/vcard-elements.xhtml)).
+    pub unknown_: UnsupportedProperties,
```

Comment:
Is there a reason that `unknown_` ends with an underscore while `extensions` does not?

Reply:
Do we even need the distinction? As a developer I'd have to think about if my key starts with "x-" so then I'd have to query `extensions` otherwise `unknown`. Unsure if it adds something.

Reply:
Yes I was also hesitating between merging the two or not and didn't because of wording. I'll merge them.

Reply:
I underscored `unknown_` because `unknown` might be a IANA-defined key in the future. At first I had prefixed `extensions` too but since it's common in other libraries and a term used in the base RFC, I figured it would never conflict. Anyway, if `unknown` ever conflicts then it won't be much of a problem since the value will end up in `"unknown"` under `.unknown`.

Reply:
If I merge `unknown_` and `extensions`, would `unknown` be a good naming choice?